logstash-core/lib/logstash/webserver.rb

```diff
@@ -8,19 +8,20 @@
 module LogStash 
   class WebServer
 
+    class HostBindingError < StandardError; end
+
     extend Forwardable
 
     attr_reader :logger, :status, :config, :options, :cli_options, :runner, :binder, :events
 
     def_delegator :@runner, :stats
 
-    DEFAULT_HOST = "127.0.0.1".freeze
```

Comment:
well, usually it should happen at most once or twice, unless you're starting dozens of LS instances in the same box